### PR TITLE
Zabbix Agent 6.4

### PR DESCRIPTION
**Pull request by mimugmail**

### CODE REVIEW

File: net-mgmt/zabbix-agent/Makefile [hunk 1]
```diff
@@ -1,9 +1,12 @@
 PLUGIN_NAME=		zabbix-agent
 PLUGIN_VERSION=		1.13
-PLUGIN_REVISION=	2
+PLUGIN_REVISION= 3
```
Comment (fichtner) [inline net-mgmt/zabbix-agent/Makefile:3]: Don't think this is needed just to add a package :)

File: net-mgmt/zabbix-agent/Makefile [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 PLUGIN_COMMENT=		Zabbix monitoring agent
 PLUGIN_MAINTAINER=	opnsense@moov.de
-PLUGIN_VARIANTS=	zabbix62 zabbix6 zabbix5
+PLUGIN_VARIANTS=	zabbix64 zabbix62 zabbix6 zabbix5
```
Comment (fichtner) [inline net-mgmt/zabbix-agent/Makefile:6]:
```suggestion
PLUGIN_VARIANTS=	zabbix6 zabbix62 zabbix64 zabbix5
```

File: net-mgmt/zabbix-agent/Makefile (hunk 1 above)
Comment (fichtner) [inline net-mgmt/zabbix-agent/Makefile:3]:
```suggestion
PLUGIN_REVISION=	2
```

**Review by fraenki — APPROVED** [review]
LGTM! Sorry for the delay, I was on vacation.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**